First suggestion (#1)
frontkom/.github: pull_request_template.md

> # References <!-- Link to the Jira task -->

Reviewer: I'd keep this a bit more low key than a `h1`. 😅 

```suggestion
**References** <!-- Link to the Jira task -->
```

Reviewer 2: Ha yes, I would like this too.

Author: lol no idea how it ended up like that

> [PROJ-XXX](https://frontkom.atlassian.net/browse/PROJ-XXX)

Reviewer: I think we should use the auto-links, so we can keep this simple:

```suggestion
PROJ-XXX
```

Author: will it really auto link though?

Reviewer: If configured yes and in on the commit text as well. :) I've set it up on several projects:
![image](https://github.com/frontkom/.github/assets/401819/dbe72b57-8ea7-4be6-8b4e-9c55955c139f)

### Changed <!-- What changed with this PR -->

### Visuals <!-- Add here some screenshots -->

### How can this be validated? <!-- Instructions for QA -->

### Pull request checklist

> - [ ] Is it possible to deploy automatically? If it requires manual deployment, did you include the steps needed to deploy?
> - [ ] Did you make sure documentation have been updated, if needed?
> - [ ] Did you include all necessary translations, if any?
> - [ ] Did you make sure to update sanitization routines if fields for storing personal information are added?

Reviewer: I like shorter texts in there, faster to read and therefore more likely ppl will actually spend 2 sec reading them. 😄 

```suggestion
- [ ] Can be deploy automatically? _If manual actions are required:_ did you describe the deploy steps?
- [ ] Is the documentation updated, if it makes sense? <!-- Check this also if no updating is needed -->
- [ ] Are necessary translations added/updated? <!-- Check this even if no translations are being changed -->
- [ ] Did you update sanitization routines, where personal information is handled?
```
